api/notd/overlap_manager.py

```diff
@@ -0,0 +1,47 @@
+import logging
+from core.queues.sqs_message_queue import SqsMessageQueue
+from core.util import chain_util
+from core.util import list_util
+from core.store.retriever import StringFieldFilter
+
+
+from notd.collection_overlap_processor import CollectionOverlapProcessor
+from notd.messages import RefreshAllCollectionOverlapsMessageContent
+from notd.messages import RefreshCollectionOverlapMessageContent
+from notd.model import GALLERY_COLLECTIONS
+from notd.store.retriever import Retriever
+from notd.store.saver import Saver
+from notd.store.schema import TokenCollectionOverlapsTable
+
+
+class OverlapManager:
```

Comment:
i think this hsould be collectionOverlapManager, given what the processor is called?